dashboard/app/models/level.rb

```diff
@@ -25,7 +25,7 @@
 class Level < ActiveRecord::Base
   belongs_to :game
   has_and_belongs_to_many :concepts
-  has_many :script_levels, dependent: :destroy
+  has_and_belongs_to_many :script_levels
```

Comment:
Hmm. Note that this allows for the possibility of ScriptLevels existing with zero related Levels. We are of course not often able to destroy levels, but this is still potentially concerning, particularly because there are still things (like those temporary convenience methods) that assume at least one level per SL.

It's probably worth adding a simple `before_destroy` line here to remove all script levels that would be left empty.